### PR TITLE
[10.0][MIG] stock_scanner_shipping

### PR DESCRIPTION
**Pull request by xavierjimenez**
Migration of the module **stock_scanner_shipping** to version 10.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Runbot warning:
```
2018-06-12 16:12:58,893 180 WARNING openerp_test odoo.addons.base.ir.ir_model: Creating the ir.model.data scanner_scenario_stock in module stock_scanner instead of stock_scanner_shipping.
```

File: stock_scanner_shipping/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    'website': 'https://odoo-community.org/',
+    'author': 'SYLEAM,'
+              'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'stock_scanner',
+    ],
+    'data': [
+        'data/Stock.scenario',
```
Comment (misern2) [inline stock_scanner_shipping/__manifest__.py:18]: Remove stock.scenario. It is already in stock_scanner module

File: stock_scanner_shipping/data/Stock.scenario [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (misern2) [inline stock_scanner_shipping/data/Stock.scenario:1]: Remove this file.

**Review by misern2 — CHANGES_REQUESTED** [review]
Please rebase and check the requested changes

**Comment by rousseldenis** [comment]
@xavierjimenez

**Pull request closed without merging**